### PR TITLE
Add new option to one-variable-per-declarations: ignore-for-loop

### LINKED ISSUE #1204: Add ignore-for-loop option to one-variable-per-declaration rule
This option ignore check `for statement`.
related: [#1191](https://github.com/palantir/tslint/pull/1191#issuecomment-216311700)

### PR DESCRIPTION
**Pull request by YuichiNukiyama**
fix #1204

### CODE REVIEW

File: test/rules/one-variable-per-declaration/all/tslint.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (JKillian) [inline test/rules/one-variable-per-declaration/all/tslint.json]: Can this be `test/rules/one-variable-per-declaration/default/tslint.json`?

File: src/rules/oneVariablePerDeclarationRule.ts [hunk 2]
```diff
@@ -40,9 +42,10 @@ class OneVariablePerDeclarationWalker extends Lint.RuleWalker {
 
     public visitForStatement(node: ts.ForStatement) {
         const initializer = node.initializer as ts.VariableDeclarationList;
+        const isIgnoreForLoop = this.hasOption(OPTION_IGNORE_FOR_LOOP);
```
Comment (JKillian) [inline src/rules/oneVariablePerDeclarationRule.ts]: `shouldIgnoreForLoop` would be a slightly better name than `isIgnoreForLoop`

**Comment by JKillian** [comment]
Thanks @YuichiNukiyama, this is a good option to have! Let me know when you make the small changes above and then we'll merge this

**Comment by YuichiNukiyama** [comment]
@JKillian Thanks for your review. done.

**Comment by adidahiya** [comment]
👍

**Pull request merged**